docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

```diff
@@ -211,3 +211,25 @@ with `cert-manager.io/v1`
 		return errors.Wrap(err, "failed setting up with a controller manager")
 	}
   ```
+
+## Required changes to have individual service accounts for controllers.
+
+1. Create a new service account such as:
+  ```yaml
+  apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: manager
+  namespace: system
```

Comment:
Seems this needs indentation or if this is the correct version, _service_account.yaml_ files needs to be changed? 
```suggestion
      namespace: system
```

Reply:
Yes - sorry I missed this. Maybe we can bundle the indentation change with a different PR. Not sure if it's worth opening a new PR.

Reply:
True, can be tackled in the series of upcoming changes for providers.  cc @fabriziopandini